pom.xml

```diff
@@ -0,0 +1,285 @@
+		<url>https://ibissource.org</url>
+	</organization>
+	<licenses>
+		<license>
+			<name>Apache License, Version 2.0</name>
+			<url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>manual</distribution>
+		</license>
+	</licenses>
+	<developers>
+		<developer>
+			<name>Martijn Dirkse</name>
+			<email>martijn@ibissource.org</email>
+			<organization>WeAreFrank!</organization>
+			<organizationUrl>https://wearefrank.nl</organizationUrl>
+		</developer>
+		<developer>
+			<!-- I copied these from the IAF repository, but is this data still correct? -->
+			<name>Niels Meijer</name>
+			<email>niels@ibissource.org</email>
+			<organization>Ibissource</organization>
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+		<developer>
+			<name>Gerrit van Brakel</name>
+			<email>gerrit@ibissource.org</email>
+			<organization>Ibissource</organization>
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+		<developer>
+			<name>Jaco de Groot</name>
+			<email>jaco@ibissource.org</email>
+			<organization>Ibissource</organization>
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+	</developers>
+	<build>
+		<defaultGoal>install</defaultGoal>
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<version>3.2.0</version>
+				<configuration>
+					<archive>
+						<addMavenDescriptor>true</addMavenDescriptor>
+						<manifest>
+							<addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+							<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+						</manifest>
+					</archive>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-source-plugin</artifactId>
+				<version>3.2.1</version>
+				<executions>
+					<execution>
+						<id>attach-sources</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>jar-no-fork</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.glassfish.copyright</groupId>
+				<artifactId>glassfish-copyright-maven-plugin</artifactId>
+				<version>2.4</version>
+				<executions>
+					<execution>
+						<?m2e ignore?> <!-- Eclipse doesn't seem to like this plugin, it doesn't need to be executed by Eclipse so ignore it -->
+						<id>Check-CopyRight-Notice</id>
+						<goals>
+							<goal>copyright</goal><!-- Checks for Copyright notices -->
+							<goal>check</goal><!-- Fails the build -->
+						</goals>
+						<phase>process-sources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<excludeFile>${project.basedir}/copyrightExcludeFile.txt</excludeFile>
```

Comment:
Houd er rekening mee dat dit alleen werkt bij een flat-project (aka geen multi-module project). 
De property `project.basedir` is relatief ten opzichte van de pom waar je de build start. Wanneer je meerdere modules hebt en één module wilt bouwen gaat het fout!

Reply:
Dank voor de tip. Ik begreep de oplossing voor multi-module projecten toen ik hem in de IAF repository zag.